src/components/ui/tooltip.tsx

```diff
@@ -84,3 +84,20 @@ export const TooltipContent = React.forwardRef<
     </FloatingPortal>
   );
 });
+
+export const WithDefaultTooltip = ({
+  children,
+  tooltip,
+}: { children: React.ReactNode; tooltip?: React.ReactNode }) => {
+  if (!tooltip) {
+    return children;
+  }
+  return (
+    <Tooltip delay={300}>
+      <TooltipTrigger asChild>{children}</TooltipTrigger>
```

Comment:
Totally agree on single child. However, I think I'd rather use a typescript solution. The `Children` API seems to have 'Legacy' status, and it just errors out on failure, think it would be better to just go around it statically.
lmk if you like this solution!

Reply:
great workaround